### PR TITLE
Update ug

### LINKED ISSUE #120: Update links, command formats, context of user guide

### PR DESCRIPTION
**Pull request by sethteo**
Fixes #120

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -226,10 +237,10 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+FinCliq data are saved automatically as a JSON file `[JAR file location]/data/fincliq.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md]: This means we have to change the name of the data file from `addressbook.json` to `fincliq.json`.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -146,44 +146,56 @@ Examples:
...
+Format: `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`
 
 - `CLIENT_INDEX` refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​.
-- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `2023-01-01 14:00`.
-- A meeting can have an optional description(s).
+- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `02-01-2025 12:00`.
+- `DESCRIPTION` refers to what the meeting is about. Format should be a string.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A meeting can have any number of descriptions (including 0).
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md]: Perhaps we need to update this line? Since we only have one description.

**Review by itstrueitstrueitsrealitsreal — CHANGES_REQUESTED** [review]
Looks good, the command formats and examples seem to be updated. Might need to change line 13-14 of `index.md` to reflect the change from AddressBook to FinCliQ.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -146,44 +146,54 @@ Examples:
...
+- `filter friends` Displays all clients with the tag `friends`
+
 ## Meeting Functions
 
 ### Adding a Meeting: `add`
 
 Adds a meeting for a specific client in the address book.
 
-Format: `add CLIENT_INDEX n/NAME dt/DATE_TIME v/VENUE [d/DESCRIPTION]…​`
+Format: `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`
```
Comment (timothysashimi) [inline docs/UserGuide.md:167]: clear statements of prefixes in example

File: docs/UserGuide.md [hunk 4]
```diff
@@ -146,44 +146,54 @@ Examples:
...
-- `list 1` Lists all meetings for the first client.
+- `view c 2` Lists all meetings of the first client.<br>
+  ![result for 'view c 2'](images/resultImages/viewClientResult.png)
 
 ### Editing a Meeting: `edit`
 
 Edits an existing meeting for a client.
 
-Format: `edit CLIENT_INDEX MEETING_INDEX [n/NAME] [dt/DATE_TIME] [v/VENUE] [d/DESCRIPTION]…​`
+Format: `editMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX n/DESCRIPTION d/DATE_TIME`
```
Comment (timothysashimi) [inline docs/UserGuide.md]: i think the datetime prefix is dt/? because there's a d/ for description alr

File: docs/UserGuide.md [hunk 5]
```diff
@@ -263,25 +274,26 @@ _Details coming soon ..._
...
 
-| Action     | Format, Examples                                                |
-| ---------- | --------------------------------------------------------------- |
-| **Add**    | `add CLIENT_INDEX n/NAME`<br>e.g., `add 1 n/James Ho`           |
-| **List**   | `list CLIENT_INDEX`<br>e.g., `list 2`                           |
-| **Edit**   | `edit CLIENT_INDEX MEETING_INDEX`<br>e.g.,`edit 1 2 n/Jamal Ho` |
-| **Delete** | `delete CLIENT_INDEX MEETING_INDEX`<br>e.g., `delete 3 1`       |
+| Action     | Format, Examples                                                                                                                                                                       |
+|------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`<br>e.g., `addMeeting client/1 dt/02-01-2025 12:00 d/Sign life plan`                                                   |
```
Comment (timothysashimi) [inline docs/UserGuide.md]: great use of table formatting

File: docs/UserGuide.md [hunk 6]
```diff
@@ -263,25 +274,26 @@ _Details coming soon ..._
...
+| **List**   | `view c CLIENT_INDEX`<br>e.g., `view c 2`                                                                                                                                              |
+| **Edit**   | `editMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX n/DESCRIPTION d/DATE_TIME`<br>e.g.,`editMeeting clientIndex/1 meetingIndex/2 n/starbucks meeting dt/01-01-2024 12:00` |
+| **Delete** | `deleteMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX`<br>e.g., `deleteMeeting clientIndex/2 meetingIndex/1`                                                              |
```
Comment (timothysashimi) [inline docs/UserGuide.md:292]: great use of asterisks to indicate importance

File: docs/UserGuide.md [hunk 7]
```diff
@@ -263,25 +274,26 @@ _Details coming soon ..._
...
 
 ### General Functions
 
 | Action    | Format, Examples |
-| --------- | ---------------- |
```
Comment (timothysashimi) [inline docs/UserGuide.md:285]: LGTM!

File: docs/UserGuide.md (hunk 3 above)
Comment (timothysashimi) [inline docs/UserGuide.md:167]: clear statement of predixes in example

Comment (chewbum) [inline docs/UserGuide.md:167]: accurate representation of the prefixes for the different commands

File: docs/UserGuide.md (hunk 5 above)
Comment (chewbum) [inline docs/UserGuide.md]: The example should be clientIndex/

File: docs/images/Ui.png
Comment (chewbum) [inline docs/images/Ui.png:1]: LGTM

**Review by chewbum — COMMENTED** [review]
Great job!

File: docs/UserGuide.md (hunk 2 above)
Comment (sethteo) [reply docs/UserGuide.md]: Let me know if the changes convey what you are trying to describe

File: docs/UserGuide.md [hunk 8]
```diff
@@ -263,25 +274,26 @@ _Details coming soon ..._
...
+|------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`<br>e.g., `addMeeting clientIndex/1 dt/02-01-2025 12:00 d/Sign life plan`                                              |
+| **List**   | `view c CLIENT_INDEX`<br>e.g., `view c 2`                                                                                                                                              |
+| **Edit**   | `editMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX n/DESCRIPTION d/DATE_TIME`<br>e.g.,`editMeeting clientIndex/1 meetingIndex/2 n/starbucks meeting dt/01-01-2024 12:00` |
+| **Delete** | `deleteMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX`<br>e.g., `deleteMeeting clientIndex/2 meetingIndex/1`                                                              |
```
Comment (jaejayrome) [inline docs/UserGuide.md:292]: Good Markdown Formatting to ensure that user guide is completed!

File: docs/UserGuide.md [hunk 9]
```diff
@@ -193,20 +203,21 @@ Format: `edit CLIENT_INDEX MEETING_INDEX [n/NAME] [dt/DATE_TIME] [v/VENUE] [d/DE
...
-Format: `delete CLIENT_INDEX MEETING_INDEX`
+Format: `deleteMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX`
 
 - Deletes the meeting specified by `MEETING_INDEX` for the client specified by `CLIENT_INDEX`. Both indexes must be positive integers 1, 2, 3, …​.
 
 Example:
 
-- `delete 3 1` Deletes the first meeting for the third client.
+- `deleteMeeting clientIndex/2 meetingIndex/1` Deletes the first meeting for the first client.<br>
+  ![delete meeting result](images/resultImages/deleteMeetingResult.png)
```
Comment (jaejayrome) [inline docs/UserGuide.md:220]: I like how you add the descriptive images to better inform users on what a successful execution of the command would look like!

**Review by jaejayrome — COMMENTED** [review]
lgtm, thanks for your pull request

**Pull request merged**